webapp/src/components/data_selectors/jira_autocomplete_selector/jira_autocomplete_selector.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 
 import {ReactSelectOption} from 'types/model';
 
-import BackendSelector, {Props as BackendSelectorProps} from '../backend_selector';
+import BackendSelector, {Props as BackendSelectorProps} from 'components/data_selectors/backend_selector';
```

Comment:
I wonder why this was suggested by the linter? I like how the relative import shows that the components are related to each other

Reply:
Yeah 2/5 let's keep relative paths in these repos